CHANGELOG.md

```diff
@@ -41,9 +41,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fixed issue where satellite to satellite matchups with the same dataset don't return the expected result
 - Fixed CSV and NetCDF matchup output bug
 - Fixed NetCDF output switching latitude and longitude
+- SDAP-399: Updated quickstart guide for standalone docker deployment of SDAP.
+- SDAP-399: Updated quickstart Jupyter notebook
 - Fixed import error causing `/timeSeriesSpark` queries to fail.
 - Fixed bug where domsresults no longer worked after successful matchup
 - Fixed certificate error in Dockerfile
-### Security
```

Comment:
you can leave this section in empty in case we make a security change

Reply:
I think I removed that line by accident. Oops

Reply:
Well this is strange...
It's there in the branch

Reply:
https://github.com/RKuttruff/incubator-sdap-nexus/blob/quickstart-update/CHANGELOG.md?plain=1#L49

Reply:
Added a newline after. It no longer shows as deleted